testData/solutions/win32.json5

```diff
@@ -1691,396 +1773,434 @@
-                            "type": "number",
-                            "default": 1,
-                            "enum": [
-                                0,
-                                1,
-                                2
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "highlight",
-                                "all"
-                            ]
+                            "title": "Error Message Verbosity",
+                            "description": "How verbose \"Error Message\" announcements should be.",
+                            "type": "string"
                         }
                     },
-                    "Options.SimultaneousSynthAndWave": {
+                    "OutputModes.HELP_BALLOON": {
                         "schema": {
-                            "title": "Simultaneous Synth and Wave",
-                            "description": "Whether to enable smoother playing of WAV files when using the DEC Talk Access 32 software synthesizer. This option is only available in Default.jcf. Changes to this setting take effect the next time you start JAWS.  Disabled by default.",
-                            "type": "number",
-                            "default": 0,
-                            "enum": [
-                                0,
-                                1
-                            ],
-                            "enumLabels": [
-                                "off",
-                                "on"
-                            ]
+                            "title": "Help Balloon Verbosity",
+                            "description": "How verbose \"Help Balloon\" announcements should be.",
+                            "type": "string"
                         }
                     },
-                    "Options.SingleDigitThreshold": {
+                    "OutputModes.HELP_MESSAGE": {
                         "schema": {
-                            "title": "Single Digit Threshold",
-                            "description": "The number of digits a number must contain before it is read as single digits (like a phone number).  Defaults to five.",
-                            "type": "integer",
-                            "default": 5
+                            "title": "Help Message Verbosity",
+                            "description": "How verbose \"Help Message\" announcements should be.",
+                            "type": "string"
```

Comment:
I'll stop commenting on these individually, but all of these string values without even a default are very low quality.

I would suggest adding a warning [to the solution registry file tests](https://github.com/GPII/universal/blob/master/gpii/node_modules/solutionsRegistry/test/SolutionsRegistryFileTests.js#L205) for any setting that lacks a default.  It'll make it easier to look for these.

Reply:
Thanks. I've updated (ie. commented out with comments) all the errors that come from the work you put in on adding warnings.